### PR TITLE
[NEXT] PR for issue 114

### PR DESCRIPTION
**Pull request by hsbasu**
This PR fixes and closes issue #114.

## Tested on:
1. Ubuntu Mate 20.04.2
2. Ubuntu 20.10
3. Ubuntu 21.04

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,67 @@
...
+			```
+			from the `/path/to/repo` in a terminal. It will create the translations/locales in `usr/share/locale`.
+		
+		2. Copy the contents of `etc` and `usr` to `/etc/` and `/usr/` respectively:
+			```
+			sudo cp -R usr etc /
+			```
+		3. Compile `schemas` using:
+			```
+			glib-compile-schemas /usr/share/glib-2.0/schemas
```
Comment (goodevilgenius) [inline README.md:51]: This might need to be run with `sudo`. I'm not sure, though.

Comment (clefebvre) [reply README.md:51]: it does

**Comment by clefebvre** [comment]
Please make separate PRs for separate fixes.

Content visible to users in the UI need to be translated with gettext.

**Comment by hsbasu** [comment]
@clefebvre Can you please tell me how to do the translation part?

**Comment by hsbasu** [comment]
Also I am not sure whether to create separate PRs, should I delete the current PR or I can separate the PR. Please help.

**Pull request closed without merging**